bundle-workflow/src/test_workflow/test_result/test_result.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+
+class TestResult:
+    def __init__(self, component, config, status):
+        self.component = component
+        self.config = config
+        self.status = status
+
+    @property
+    def __test_result(self):
+        return "PASS" if self.status == 0 else "FAIL"
+
+    def __str__(self):
+        return f'| {self.component} | {self.config} | {self.__test_result} | {self.status} |'
```

Comment:
nit: something like this will help keep the alignment correct in the table. (assuming every variable is a string) 
```suggestion
        return '| {:20s} | {:15s} | {:5s} | {:4s} |'.format(self.component, self.config, self.__test_result, self.status)
```

e.g.
```
2021-10-01 21:25:12 INFO  | alerting            | without-security | PASS | 0 |
2021-10-01 21:25:12 INFO  | anomaly-detection   | with-security    | PASS | 0 |
```